pkg/symbolizer/symbolizer.go

```diff
@@ -67,6 +67,8 @@ func (s *Symbolizer) getSubprocess(bin string) (*subprocess, error) {
 	if sub := s.subprocs[bin]; sub != nil {
 		return sub, nil
 	}
+	// TODO(HerrSpace): Check if binutils was build with macho support if we
```

Comment:
How hard is it to check? If it's not too complicated, looks reasonable to resolve now.

Reply:
True. Supported targets are printed in `addr2line --help`

Reply:
Implemented.